aidatlu/test/fixtures/tlu_test_configuration.yaml

```diff
@@ -63,7 +63,7 @@ pmt_control:
 # Save and generate interpreted data from the raw data set. Set to 'True' or 'False'.
 # If no specific output path is provided, the data is saved in the default output data path (aidatlu/aidatlu/tlu_data).
 save_data: True
-output_data_path: 'test_output_data/'
+output_data_path: 'aidatlu/test/fixtures/test_output_data/'
```

Comment:
I am not 100% sure, but `pytest` looks for test in all directories that start with `test`. Also, I do not know the best practice where to store such output files temporarily, I would not expect them to be part of the fixtures, but that is fine for me at the moment.

Reply:
Feel free to resolve it, this should not be part of the review, but a general thought I had that we can discuss at some point or just leave it be.